extensions/wikia/WhereIsExtension/templates/list.tmpl.php

```diff
@@ -197,7 +208,7 @@ function(e) {
 		$.ajax({
 			type:"POST",
 			dataType: "json",
-			url: ajaxpath+'?action=ajax&rs=axWFactoryFilterVariables' + values,
+			url: ajaxpath + encodeURIComponent('?action=ajax&rs=axWFactoryFilterVariables' + values),
```

Comment:
`encodeURIComponent` here will break the URL as it will encode the `?` and so forth. It should instead be used on the values being set earlier and passed here, i.e.
```javascript
		values += '&group=' + encodeURIComponent($('#groupSelect').val());
		values += '&string=' + encodeURIComponent($('#withString').val());
```